tests/test_schema.py

```diff
@@ -2,21 +2,27 @@
 
 import jsonschema
 import pyarrow as pa
-import pytest
 
 from meds import (
-    data_schema, label_schema, dataset_metadata_schema, patient_split_schema, code_metadata_schema,
-    train_split, tuning_split, held_out_split
+    code_metadata_schema,
+    data_schema,
+    dataset_metadata_schema,
+    held_out_split,
+    label_schema,
+    subject_split_schema,
+    train_split,
+    tuning_split,
 )
 
+
 def test_data_schema():
     """
     Test that mock data follows the data schema.
     """
     # Each element in the list is a row in the table
     raw_data = [
         {
-            "patient_id": 123,
+            "subject_id": 123,
```

Comment:
I'd recommend importing and using the dedicated constants where possible here, but feel free to ignore that. I know that there are arguments both ways for that in the context of tests.

Reply:
I think it would be best to keep the test as is for simplicity.